LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2023 Daishi Kato
+Copyright (c) 2025 Daishi Kato
```

Comment:
Copyright year should be the first publication year. So, I don't think we need to update this.